volatility3/framework/layers/physical.py

```diff
@@ -113,14 +113,15 @@ def _file(self) -> IO[Any]:
     def maximum_address(self) -> int:
         """Returns the largest available address in the space."""
         # Zero based, so we return the size of the file minus 1
-        if self._size:
-            return self._size
+        if self._maximum_address
```

Comment:
Is it valid without `:` syntax for `if` statements?
```suggestion
        if self._maximum_address:
```

Reply:
fixed it

Reply:
Thank you always for your quick response. :)